src/neuronmusic/CMakeLists.txt

```diff
@@ -82,37 +81,7 @@ fi\n\
   endforeach(pyexe)
 
   add_dependencies(neuronmusicextension nrniv_lib neuronmusic_cython_generated)
-
-  # ~~~
-  # (Copied from src/nrnpython/CMakeLists.txt)
-  # neuron module (possibly with multiple extension versions) was built
-  # in NRN_PYTHON_BUILD_LIB. Not a problem if install overwrites multiple
-  # times to same install folder or if each install ends up in different
-  # place.
-  # ~~~
-  file(
-    WRITE ${CMAKE_CURRENT_BINARY_DIR}/neuronmusic_module_install.sh
-    "\
-#!bash\n\
-echo 'Installing python module using:'\n\
-set -ex\n\
-cd ${CMAKE_CURRENT_BINARY_DIR}\n\
-export SETUPTOOLS_USE_DISTUTILS=stdlib
-$1 setup.py --quiet build --build-lib=${NRN_PYTHON_BUILD_LIB} install ${NRN_MODULE_INSTALL_OPTIONS}\n\
-")
-  foreach(pyexe ${NRN_PYTHON_EXE_LIST})
-    # install(CODE ...) only takes a single CMake code expression, so we can't easily roll our own
-    # check on the return code. Modern CMake versions support the COMMAND_ERROR_IS_FATAL option,
-    # which will cause the installation to abort if the shell script returns an error code.
-    if(${CMAKE_VERSION} VERSION_LESS "3.19")
-      install(
-        CODE "execute_process(COMMAND bash ${CMAKE_CURRENT_BINARY_DIR}/neuronmusic_module_install.sh ${pyexe})"
-      )
-    else()
-      install(
-        CODE "execute_process(COMMAND bash ${CMAKE_CURRENT_BINARY_DIR}/neuronmusic_module_install.sh ${pyexe} COMMAND_ERROR_IS_FATAL LAST)"
-      )
-    endif()
-  endforeach(pyexe)
```

Comment:
have not much insight on music side cc: @nrnhines

Reply:
Unfortunately I haven't checked if this works or not (since we don't have any test)

Reply:
so there are tests, but master did not work on my Ubuntu. Created a PR for fix & coverage: https://github.com/neuronsimulator/nrn/pull/2209

Reply:
Hitting some issues with #2209, I cannot help but feel things are not finalized when it comes to MUSIC? building master on Ubuntu didn't work and now in the coverage build I'm experiencing new issues. @nrnhines are you ok with merging this PR as-is?

Reply:
@pramodk @nrnhines I'd hate for this to be "stuck" on MUSIC feature as I am also working on having one single setup.py. Since no tests are currently setup for master and we still have quite a few things to workout for MUSIC ( see #2092) I suggest we merge this and tackle whatever might be required  afterwards.

Reply:
With the fix from #2209 tests pass locally: 
```
76: Test timeout computed to be: 300
1/1 Test #76: nrnmusic::music_tests ............   Passed    1.46 sec

The following tests passed:
        nrnmusic::music_tests

100% tests passed, 0 tests failed out of 1

Total Test time (real) =   1.47 sec
```

Reply:
Ok now we have CI via 824c7652b037a2f1fb8d9d4d0a84391973846678

Reply:
Let's go with this.